docs/developer/app-store/apps/index.mdx

```diff
@@ -0,0 +1,29 @@
+import CardGrid from "@site/components/CardGrid";
+
+# Apps
```

Comment:
I'd rephrase to `Appstore Apps` or something like this.

Keep in mind that its SEO-indexed and people will see this as a first page, not after clicking "app-store" section in the menu. 

Apps have several concepts in Saleor:
1. Webhooks / simple local apps
2. 3rd party / custom apps & their development
3. Using Appstore & apps provided by Saleor
4. Forking/working with Saleor Apps

All of them are separate topics, connected, but often exclusive